// poe-api-manager/dist/lib/modules/poe.watch/allModules/Modules.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.WeaponModule = exports.UniqueMapsModule = exports.SextantModule = exports.ScarabModule = exports.OilModule = exports.InvitationModule = exports.InscribedModule = exports.GemModule = exports.JewelModule = exports.FlaskModule = exports.MapModule = exports.FragmentModule = exports.FossilModule = exports.EssenceModule = exports.DivsModule = exports.DeliriumOrbModule = exports.BeastModule = exports.BaseModule = exports.ArmourModule = exports.AccessoryModule = exports.CurrencyModule = void 0;
const CurrencyModule_1 = __importDefault(require("./subModules/CurrencyModule"));
exports.CurrencyModule = CurrencyModule_1.default;
const AccessoryModule_1 = __importDefault(require("./subModules/AccessoryModule"));
exports.AccessoryModule = AccessoryModule_1.default;
const ArmourModule_1 = __importDefault(require("./subModules/ArmourModule"));
exports.ArmourModule = ArmourModule_1.default;
const BaseModule_1 = __importDefault(require("./subModules/BaseModule"));
exports.BaseModule = BaseModule_1.default;
const BeastModule_1 = __importDefault(require("./subModules/BeastModule"));
exports.BeastModule = BeastModule_1.default;
const DeliriumOrbModule_1 = __importDefault(require("./subModules/DeliriumOrbModule"));
exports.DeliriumOrbModule = DeliriumOrbModule_1.default;
const DivsModule_1 = __importDefault(require("./subModules/DivsModule"));
exports.DivsModule = DivsModule_1.default;
const EssenceModule_1 = __importDefault(require("./subModules/EssenceModule"));
exports.EssenceModule = EssenceModule_1.default;
const FossilModule_1 = __importDefault(require("./subModules/FossilModule"));
exports.FossilModule = FossilModule_1.default;
const FragmentModule_1 = __importDefault(require("./subModules/FragmentModule"));
exports.FragmentModule = FragmentModule_1.default;
const MapModule_1 = __importDefault(require("./subModules/MapModule"));
exports.MapModule = MapModule_1.default;
const FlaskModule_1 = __importDefault(require("./subModules/FlaskModule"));
exports.FlaskModule = FlaskModule_1.default;
const JewelModule_1 = __importDefault(require("./subModules/JewelModule"));
exports.JewelModule = JewelModule_1.default;
const GemModule_1 = __importDefault(require("./subModules/GemModule"));
exports.GemModule = GemModule_1.default;
const InscribedModule_1 = __importDefault(require("./subModules/InscribedModule"));
exports.InscribedModule = InscribedModule_1.default;
const InvitationModule_1 = __importDefault(require("./subModules/InvitationModule"));
exports.InvitationModule = InvitationModule_1.default;
const OilModule_1 = __importDefault(require("./subModules/OilModule"));
exports.OilModule = OilModule_1.default;
const ScarabModule_1 = __importDefault(require("./subModules/ScarabModule"));
exports.ScarabModule = ScarabModule_1.default;
const SextantModule_1 = __importDefault(require("./subModules/SextantModule"));
exports.SextantModule = SextantModule_1.default;
const UniqueMapsModule_1 = __importDefault(require("./subModules/UniqueMapsModule"));
exports.UniqueMapsModule = UniqueMapsModule_1.default;
const WeaponModule_1 = __importDefault(require("./subModules/WeaponModule"));
exports.WeaponModule = WeaponModule_1.default;
